app/services/health_checks/gov_uk_notify.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module HealthChecks
+  class GovUkNotify
+    URL = 'https://status.notifications.service.gov.uk'
```

Comment:
how about a default:
```
 ENV.fetch('GOV_UK_NOTIFY_STATUS_URL', 'https://status.notifications.service.gov.uk')
```
then if it is not set (e.g. in a dev environment) it will still work ?